altair/vegalite/v5/_api_rfc.py

```diff
@@ -0,0 +1,372 @@
+        return _parse_aggregate("q1", col_name, type)
+
+    @classmethod
+    def q3(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("q3", col_name, type)
+
+    @classmethod
+    def ci0(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("ci0", col_name, type)
+
+    @classmethod
+    def ci1(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("ci1", col_name, type)
+
+    @classmethod
+    def stderr(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("stderr", col_name, type)
+
+    @classmethod
+    def stdev(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("stdev", col_name, type)
+
+    @classmethod
+    def stdevp(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("stdevp", col_name, type)
+
+    @classmethod
+    def sum(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("sum", col_name, type)
+
+    @classmethod
+    def valid(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("valid", col_name, type)
+
+    @classmethod
+    def values(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("values", col_name, type)
+
+    @classmethod
+    def variance(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("variance", col_name, type)
+
+    @classmethod
+    def variancep(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("variancep", col_name, type)
+
+    @classmethod
+    def exponential(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("exponential", col_name, type)
+
+    @classmethod
+    def exponentialb(
+        cls, col_name: str | None = None, /, type: EncodeType = None
+    ) -> dict[str, Any]:
+        return _parse_aggregate("exponentialb", col_name, type)
+
+
+class field:
+    """
+    Utility class for field predicates and shorthand parsing.
+
+    Examples
+    --------
+    >>> field("Origin")
+    {'field': 'Origin'}
+
+    >>> field("Origin:N")
+    {'field': 'Origin', 'type': 'nominal'}
+
+    >>> field.one_of("Origin", "Japan", "Europe")
+    SelectionPredicateComposition({'field': 'Origin', 'oneOf': ['Japan', 'Europe']})
+    """
+
+    def __new__(  # type: ignore[misc]
```

Comment:
Wanted to highlight this current design quirk.

```py
import altair as alt

>>> alt.field("Origin:N")
{'field': 'Origin', 'type': 'nominal'}

>>> alt.agg("Origin:N")
{'field': 'Origin', 'type': 'nominal'}
```

1. I'm anticipating users would expect `alt.field()` to work this way, despite shorthand being unrelated.
2. I also think using `alt.agg()` without specifying an aggregation wouldn't seem an intuitive way to define a field.
3. Each wrapped [`Field...Predicate`](https://github.com/vega/altair/blob/679a7ce30cb3366319bce2f4211dd62a5a80c87c/altair/vegalite/v5/schema/core.py#L16401) already has a constructor; the faux-parent [`Predicate`](https://github.com/vega/altair/blob/679a7ce30cb3366319bce2f4211dd62a5a80c87c/altair/vegalite/v5/schema/core.py#L16392) class is just a `Union` and the unrelated [`Field`](https://github.com/vega/altair/blob/679a7ce30cb3366319bce2f4211dd62a5a80c87c/altair/vegalite/v5/schema/core.py#L7353) class